src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -513,6 +513,7 @@ def get(self, request, organization, trace_id):
         return Response(self.serialize(result["data"][0]))
 
     def serialize(self, results):
+        return results
```

Comment:
Is this a typo?

Reply:
.... 🤦 yes this was me checking that its null when there's no results